stdeb.cfg

```diff
@@ -1,7 +1,7 @@
 [rospkg]
 Debian-Version: 100
-Depends: python-catkin-pkg, python-rospkg-modules
-Depends3: python3-catkin-pkg, python3-rospkg-modules
+Depends: python-argparse, python-catkin-pkg, python-rospkg-modules
+Depends3: python-argparse, python3-catkin-pkg, python3-rospkg-modules
```

Comment:
Please remove the change for Python 3. First it can't depend on the Python 2 version of argparse. Second `argparse` is part of Python 3 and ha no separate Debian package.

Reply:
See d9f2760a21842f791ceabaa5c4cae8082fdc5968.